lib/src/components/workflowStep/workflowStepCollapsibleBody.js

```diff
@@ -2,21 +2,23 @@ import React, { useContext, useEffect, useRef, useState } from 'react';
 import { useTransition, animated } from 'react-spring';
 import * as easings from 'd3-ease';
 import { WorkflowStepContext } from './workflowStepProvider';
+import { useSize } from '../../../helpers/useSize';
 
 export function WorkflowStepCollapsibleBody({ children }) {
   const { showBody } = useContext(WorkflowStepContext);
   const bodyRef = useRef(null);
-  const [shouldAllowOverflow, setShouldAllowOverflow] = useState(false);
+  const size = useSize(bodyRef);
```

Comment:
We need to have the correct up-to-date value on the height of our collapsible body. This is a hook I found on the internets that gives us that.

I tried writing one myself and it turns out it's not that simple!